packages/material-ui-lab/src/Slider/Slider.js

```diff
@@ -463,7 +485,9 @@ class Slider extends React.Component {
             onTouchStartCapture={this.handleTouchStart}
             onTouchMove={this.handleMouseMove}
             onFocusVisible={this.handleFocus}
-          />
+          >
+            {Thumbnail()}
```

Comment:
I would change the `Thumbnail` assignment so that you don't have to use this call syntax here. It's more in line with other components and also removes a second call to the function.

Reply:
Fixed.